types/node/v7/index.d.ts

```diff
@@ -1583,9 +1583,12 @@ declare module "readline" {
         completer?: Completer | AsyncCompleter;
         terminal?: boolean;
         historySize?: number;
+        prompt?: string;
+        crlfDelay?: number;
+        removeHistoryDuplicates?: boolean;
     }
 
-    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean): ReadLine;
+    export function createInterface(input: NodeJS.ReadableStream, output?: NodeJS.WritableStream, completer?: Completer | AsyncCompleter, terminal?: boolean, historySize?: number, prompt?: string, crlfDelay?: number, removeHistoryDuplicates?: boolean): ReadLine;
```

Comment:
Hmm, I have been able to successfully use createInterface with 8 parameters in the past. However, I am OK with this edit, its easy enough to use the ReadLineOptions format. This is my first time contributing, do I need to do anything further?